flink-end-to-end-tests/test-scripts/elasticsearch-common.sh

```diff
@@ -56,13 +56,14 @@ function verify_elasticsearch_process_exist {
 
 function verify_result {
     local numRecords=$1
+    local index=$2
```

Comment:
adjust existing usages in `test_streaming_elasticsearch.sh`

Reply:
+1